src/main/resources/static/api.yml

```diff
@@ -1366,6 +1366,57 @@ paths:
           $ref: '#/components/responses/403Response'
         500:
           $ref: '#/components/responses/500Response'
+  /assessments/{assessmentId}/acceptances:
```

Comment:
I think `  /assessments/{assessmentId}/acceptance` would be more appropriate, as there will only be one `acceptance` per `assessment`